app/code/Magento/Customer/Block/Account/Delimiter.php

```diff
@@ -15,8 +16,7 @@
 class Delimiter extends \Magento\Framework\View\Element\Template implements SortLinkInterface
 {
     /**
-     * {@inheritdoc}
-     * @since 101.0.0
```

Comment:
Why `@since` annotation has been removed?

Reply:
corrected that